src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java

```diff
@@ -26,6 +26,22 @@ public AmountContentProvider(InterestContentProvider interestContentProvider) {
         this.interestContentProvider = interestContentProvider;
     }
 
+    private BigDecimal getPartAdmissionAmount(Claim claim, PartAdmissionResponse response) {
+        return response.getAmount().subtract(
+            claim.getClaimData().getFeesPaidInPound())
```

Comment:
doesn't look related to your PR but can we rename `getFeesPaidInPound()` too with `...Pounds` - and is it plural fees?

Reply:
I can confirm that the requested change has been done properly